hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -180,7 +180,17 @@
   //  the field and value should be module_inst.signal_name
```

Comment:
btw, do you think we should change the comment from `signal_name` to `port_name`?

Reply:
actually nvm...especially top nets, i guess it could be module port, top level port, or top level signal now.

Reply:
Good.